game.js

```diff
@@ -0,0 +1,84 @@
+window.onload = function(){
+
+  var Price = function() {
+      this.firstDigit = Math.floor(Math.random()*9)+1;
+      this.secondDigit = Math.floor(Math.random()*10);
+      this.thirdDigit = Math.floor(Math.random()*10);
+      this.dishwasherValue = function() {
+        return this.thirdDigit.toString() + this.secondDigit.toString() + this.firstDigit.toString();
+
+    }
+    this.show = function(){
+      this.firstGuess = document.getElementById('digit-one').value;
+      this.secondGuess = document.getElementById('digit-two').value;
+      this.thirdGuess = document.getElementById('digit-three').value;
+    }
+  }
+
+  document.getElementById('start-button').onclick = function() {
+    var newPrice = new Price();
+    newPrice.dishwasherValue();
+    newPrice.show();
+    console.log(newPrice.firstDigit);
+    var startHint, firstHint, secondHint, yourGuess1, yourGuess2, nextHint, lastHint, yourGuess3, finalMessage, endGame;
+
+    if (newPrice.firstDigit > 5) {
+      startHint = "The first digit it greater than 5. So, what do you think it is?";
+    }
+    else {
+      startHint = "The first digit is less than or equal to 5. So, what do you think it is?";
+    }
+
+  firstHint = document.getElementById('first-hint');
+  firstHint.textContent = startHint;
+
+  document.getElementById('input-guess1').onclick = function (){
```

Comment:
Good job on get input! This works, but getEventListener('click', function() {}); is the preferred way. At least until you learn jQuery ;)